_i18n/nb-no/resources/user-guides/mine-to-pool.md

```diff
@@ -0,0 +1,194 @@
+Xerus (16.04). De kan fungere på andre Ubuntu-versjoner og på andre distribusjoner, men det er ikke garantert.
+
+Disse binærfilene er tilgjengelig på [GitHub release-siden](https://github.com/xmrig/xmrig/releases/latest).
+
+Bla ned til du ser `xmrig-VERSION-xenial-x64.tar.gz`. Last ned denne filen og pakk ut arkivene til et sted du husker, som f.eks. skrivebordet ditt.
+
+### Andre Linux-distribusjoner
+
+Brukere av andre Linux-distribusjoner kan kompilere XMRig fra bunnen av. Først må avhengighetsfilene installeres:
+
+```
+# For Debian-baserte distroer
+sudo apt install \
+	build-essential \
+	cmake \
+	git \
+	libhwloc-dev \
+	libssl-dev \
+	libuv1-dev
+```
+
+Last ned XMRig-kildekoden:
+
+```
+git clone https://github.com/xmrig/xmrig.git
+cd xmrig
+```
+
+Konfigurer og kompiler XMRig:
+
+```
+cmake -Bbuild
+make -Cbuild -j$(nproc)
+```
+
+Kopier binærfilene og konfigurasjonseksempelet til hjemmemappen din:
+
+```
+cp build/xmrig ~/
+cp src/config.json ~/
+```
+
+### macOS Build
+
+Installer først XCode og [Homebrew](https://brew.sh).
+
+Bruk Homebrew til å installere avhengighetsfiler:
+
+```
+brew install \
+	cmake \
+	hwloc \
+	libmicrohttpd \
+	libuv \
+	openssl
+```
+
+Last ned XMRig-kilder:
+
+```
+git clone https://github.com/xmrig/xmrig.git
+cd xmrig
+```
+
+Konfigurer og kompiler XMRig:
+
+```
+cmake -Bbuild -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl
+make -Cbuild -j$(nproc)
+```
+
+Kopier binærfilene og konfigurasjonseksempelet til hjemmemappen din:
+
+```
+cp build/xmrig ~/
+cp src/config.json ~/
+```
+
+## Å velge en pool
+
+Det er mange pools å velge blant. Du kan finne en liste på [miningpoolstats.stream/monero](https://miningpoolstats.stream/monero).
+
+Å velge en større pool betyr at du vil se hyppigere (men mindre) utbetalinger, men å velge en mindre pool holder nettverket desentralisert. [Utvinnere vil ikke miste inntekt ved å utvinne på en mindre pool](https://redd.it/g6uh2l).
+
+## Konfigurering av utvinneren
+
+Naviger til nettsiden til din valgte pool og les dokumentasjonen deres. De bør nevne en adresse og port som du kan legge inn i utvinneren din som f.eks. `pool.xmr.pt:3333`.
+
+Etter det, åpner du config.json-filen som du kopierte eller pakket ut tidligere i tekstbehandleren din. Bla nedover til linjene hvor det står `donate.v2.xmrig.com:3333` og endre teksten på innsiden av anførselstegnene til adressen til poolen din. Linjen under bør inneholde `YOUR_WALLET_ADDRESS`. Endre den til den faktiske lommebokadressen din.
+
+Etter disse endringene har blitt gjort, bør konfigurasjonen din se ut noe som dette:
+
+```
+{
+	// [...]
+	"pools": [
+		{
+			"url": "pool.xmr.pt:3333",
+			"user": "43YjW8SZov..."
+		}
+	],
+	// [...]
+}
+```
+
+## Å starte utvinneren
+
+Windows-brukere kan dobbeltklikke på xmrig.exe. Brukere til andre operativsystemer bør bruke `cd`-kommandoen til de kommer i mappeen som inneholder XMRig, og deretter taste inn `./xmrig` etterfulgt av enter.
```

Comment:
Change to: ... bør bruke `cd`-kommandoen til de ender opp i mappen som inneholder XMRig-filen, og deretter taste inn `./xmrig` etterfulgt av enter.

Reply:
Fixed